articles/app-service-web/web-sites-create-web-app-using-vscode.md

```diff
@@ -17,116 +17,59 @@ ms.date: 02/26/2016
 ms.author: cephalin
 
 ---
-# Create an ASP.NET 5 web app in Visual Studio Code
+# Create an ASP.NET Core web app in Visual Studio Code
 ## Overview
-This tutorial shows you how to create an ASP.NET 5 web app using [Visual Studio Code (VS Code)](http://code.visualstudio.com//Docs/whyvscode) and deploy it to [Azure App Service](../app-service/app-service-value-prop-what-is.md). 
+This tutorial shows you how to create an ASP.NET Core web app using [Visual Studio Code (VS Code)](http://code.visualstudio.com//Docs/whyvscode) and deploy it to [Azure App Service](../app-service/app-service-value-prop-what-is.md). 
 
 > [!NOTE]
 > Although this article refers to web apps, it also applies to API apps and mobile apps. 
 > 
 > 
 
-ASP.NET 5 is a significant redesign of ASP.NET. ASP.NET 5 is a new open-source and cross-platform framework for building modern cloud-based web apps using .NET. For more information, see [Introduction to ASP.NET 5](http://docs.asp.net/en/latest/conceptual-overview/aspnet.html). For information about Azure App Service web apps, see [Web Apps Overview](app-service-web-overview.md).
+ASP.NET Core is a significant redesign of ASP.NET. ASP.NET Core is a new open-source and cross-platform framework for building modern cloud-based web apps using .NET. For more information, see [Introduction to ASP.NET Core](http://docs.asp.net/en/latest/conceptual-overview/aspnet.html). For information about Azure App Service web apps, see [Web Apps Overview](app-service-web-overview.md).
```

Comment:
while you're at it remove /en from the link
http://docs.asp.net/latest/conceptual-overview/aspnet.html